src/mango/test/20-no-timeout-test.py

```diff
@@ -18,15 +18,22 @@
 class LongRunningMangoTest(mango.DbPerClass):
     def setUp(self):
         self.db.recreate()
+        self.db.create_index(["value"])
         docs = []
         for i in range(100000):
-            docs.append({"_id": str(i), "another": "field"})
-            if i % 20000 == 0:
+            docs.append({"_id": str(i), "another": "field", "value": i})
+            if i % 1000 == 0:
```

Comment:
This change seems a bit awkward. Is it a performance issue with building the view or with making the updates?

Reply:
CouchDB on FDB cannot handle 20000 docs being updated in the same transaction in _bulk_docs

Reply:
Ah, good point!